indexer/packages/v4-protos/src/codegen/klyraprotocol/yields/query.ts

```
@@ -0,0 +1,208 @@
+import * as _m0 from "protobufjs/minimal";
+import { DeepPartial } from "../../helpers";
+/** GetSDAIPriceRequest is a request type for the GetSDAIPrice RPC method. */
+
+export interface GetSDAIPriceQueryRequest {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)